test/gtest/graphapi_mha_cpp_common.hpp

```diff
@@ -0,0 +1,444 @@
+    {
+
+        if(name == "OP_MATMUL")
+        {
+            assert(in_tensors.size() == 2);
+            assert(out_tensors.size() == 1);
+
+            auto* mm_desc =
+                mAlloc.allocate(gr::MatmulBuilder().setComputeType(miopenFloat8).build());
+            mGraphBuilder->addNode(mAlloc.allocate(gr::OperationMatmulBuilder{}
+                                                       .setA(in_tensors[0])
+                                                       .setB(in_tensors[1])
+                                                       .setC(out_tensors[0])
+                                                       .setMatmulDescriptor(mm_desc)
+                                                       .build()));
+        }
+        else if(name == "OP_POINTWISE:MUL")
+        {
+
+            auto* pw = makePointWiseDesc(MIOPEN_POINTWISE_MUL);
+            addBinaryPointwiseNode(pw, in_tensors, out_tensors);
+        }
+        else if(name == "OP_POINTWISE:SUB")
+        {
+            auto* pw = makePointWiseDesc(MIOPEN_POINTWISE_SUB);
+            addBinaryPointwiseNode(pw, in_tensors, out_tensors);
+        }
+        else if(name == "OP_POINTWISE:EXP")
+        {
+            auto* pw = makePointWiseDesc(MIOPEN_POINTWISE_EXP);
+            addUnaryPointwiseNode(pw, in_tensors, out_tensors);
+        }
+        else if(name == "OP_POINTWISE:RECIPROCAL")
+        {
+            auto* pw = makePointWiseDesc(MIOPEN_POINTWISE_RECIPROCAL);
+            addUnaryPointwiseNode(pw, in_tensors, out_tensors);
+        }
+        else if(name == "OP_REDUCTION:MAX")
+        {
+            addReductionNode(MIOPEN_REDUCE_TENSOR_MAX, in_tensors, out_tensors);
+        }
+        else if(name == "OP_REDUCTION:ADD")
+        {
+            addReductionNode(MIOPEN_REDUCE_TENSOR_ADD, in_tensors, out_tensors);
+        }
+        else if(name == "OP_RNG")
+        {
+            auto* rng_desc = mAlloc.allocate(gr::RngBuilder{}
+                                                 .setDistribution(MIOPEN_RNG_DISTRIBUTION_BERNOULLI)
+                                                 .setBernoulliProb(mProbDropout)
+                                                 .build());
+
+            assert(in_tensors.size() == 2); // first is seed tensor, second is offset
+            assert(out_tensors.size() == 1);
+
+            mGraphBuilder->addNode(mAlloc.allocate(gr::OperationRngBuilder{}
+                                                       .setRng(rng_desc)
+                                                       .setSeed(in_tensors[0])
+                                                       .setOffset(in_tensors[1])
+                                                       .setOutput(out_tensors[0])
+                                                       .build()));
+        }
+        else
+        {
+            MIOPEN_LOG_FATAL("Unknown graph node type: " << name);
```

Comment:
```suggestion
            FAIL() << "Unknown graph node type: " << name;
```
`addNode` returns `void`